modules/rostests/apitests/win32nt/ntuser/NtUserSystemParametersInfo.c

```diff
@@ -609,9 +611,11 @@ Test_SPI_SETDESKWALLPAPER(void)
     /* Get Wallpaper, too small buffer  */
     szNew[0] = 0; szNew[1] = 0; szNew[2] = 0;
     TEST(NtUserSystemParametersInfo(SPI_GETDESKWALLPAPER, 3, szNew, 0) == 1);
+#if 0 // This is broken
```

Comment:
```suggestion
#if 0 // FIXME: This is broken
```
(so that we can grep for the fixmes.)

Reply:
done